specification/maps/data-plane/Creator/preview/2022-09-01-preview/features.json

```diff
@@ -0,0 +1,1074 @@
+          "204": {
+            "description": "Feature Deleted."
+          },
+          "default": {
+            "$ref": "../../../Common/preview/2022-09-01-preview/common.json#/responses/default"
+          }
+        }
+      }
+    }
+  },
+  "definitions": {
+    "LandingPageResult": {
+      "type": "object",
+      "required": [
+        "links"
+      ],
+      "properties": {
+        "ontology": {
+          "$ref": "../../../Common/preview/2022-09-01-preview/common.json#/definitions/Ontology"
+        },
+        "links": {
+          "description": "Links to other Features endpoints.",
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/FeaturesEndpointLink"
+          }
+        }
+      }
+    },
+    "ApiDefinitionResult": {
+      "type": "object",
+      "required": [
+        "openapi",
+        "info",
+        "paths",
+        "components",
+        "tags"
+      ],
+      "properties": {
+        "openapi": {
+          "description": "The OpenAPI version.",
+          "type": "string",
+          "example": "3.0.1"
+        },
+        "info": {
+          "description": "The OpenAPI info object.",
+          "type": "object",
+          "properties": {
+            "title": {
+              "description": "The title of the API.",
+              "type": "string",
+              "example": "Azure Maps Web Feature Service"
+            },
+            "version": {
+              "description": "The version of the API.",
+              "type": "string",
+              "example": "2022-09-01-preview"
+            },
+            "description": {
+              "description": "The description of the API.",
+              "type": "string",
+              "example": "Azure Maps Web Feature REST APIs"
+            }
+          }
+        },
+        "paths": {
+          "description": "The OpenAPI paths object.",
+          "type": "object"
+        },
+        "components": {
+          "description": "The OpenAPI components object.",
+          "type": "object",
+          "properties": {
+            "parameters": {
+              "description": "The OpenAPI parameters object.",
+              "type": "object"
+            },
+            "schemas": {
+              "description": "The OpenAPI schemas object.",
+              "type": "object"
+            }
+          }
+        },
+        "tags": {
+          "description": "The OpenAPI tags object.",
+          "type": "array",
+          "items": {
+            "type": "object",
+            "properties": {
+              "name": {
+                "description": "The name of the tag.",
+                "type": "string",
+                "example": "Features"
+              },
+              "description": {
+                "description": "The description of the tag.",
+                "type": "string",
+                "example": "Access to data (features)."
+              }
+            }
+          }
+        }
```

Comment:
Most times tags are just an object with `additionalProperties: true` e.g.,

```json
{
  "tags": {
    "foo": "1",
    "bar": "b"
  }
}
```

Unless this is conforming to a standard? Also, where is the value? You only document a name and description, or is this more like metadata and instances will have a version?

Also seems like `name` should be required.

Reply:
So this is the response definition for the GET /api, which is a bit interesting to document as it's returning the API definition (swagger) of the Features service. This API is required by the OGC standard as described here: https://docs.ogc.org/is/17-069r4/17-069r4.html#_api_definition_2. The tags are a list of metadata, and thanks for catching the name key being required. I've updated that portion.

Reply:
Thanks for clarifying.